packages/react/src/components/CTA/CTA.js

```diff
@@ -26,11 +33,46 @@ const { prefix } = settings;
  * @param {string} props.customClassName custom classname from parent
  * @returns {*} CTA component
  */
-const CTA = ({ style, type, customClassName, ...otherProps }) => (
-  <div className={customClassName}>
-    {renderCTA({ style, type, ...otherProps })}
-  </div>
-);
+const CTA = ({ style, type, customClassName, ...otherProps }) => {
+  const [renderLightBox, openLightBox] = useState(false);
+  const [videoTitle, setVideoTitle] = useState([{ title: '', key: 0 }]);
+  const [mediaData, setMediaData] = useState({});
+
+  useEffect(() => {
+    video();
+  }, [video, style, type]);
+
+  const video = useCallback(async () => {
```

Comment:
The const `video` seems pretty generic, this method seems to be making the call to get the meta data then setting the title. Perhaps rename to `getVideoData` or something along those lines?

Reply:
yep gotcha